mastracode/web/ui/App.tsx

```
@@ -0,0 +1,575 @@
+import { useEffect, useMemo, useRef, useState } from 'react';
+
+import { matchCommands, SLASH_COMMANDS } from './commands';
+import { GoalPanel, StatusLine, Transcript } from './components';
+import { CommandPalette } from './CommandPalette';
+import { SettingsPanel } from './SettingsPanel';
+import { ShortcutsOverlay } from './ShortcutsOverlay';
+import { applyDensity, applyTheme, loadDensity, loadTheme, saveDensity, saveTheme } from './theme';
+import type { Density, Theme } from './theme';
+import type { SlashCommand } from './commands';
+import { ChevronIcon, GearIcon, LogoMark, MenuIcon, MoonIcon, SendIcon, StopIcon, SunIcon } from './icons';
```

ESLint

[error] `./icons` import should occur before import of `./SettingsPanel`

(import/order)